app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/fraudprevention/FraudPreventionNavLinkHelper.scala

```diff
@@ -37,7 +37,7 @@ trait FraudPreventionNavLinkHelper {
     if (fraudPreventionConfig.enabled) {
       val apis            = fraudPreventionConfig.apisWithFraudPrevention
       val isProduction    = application.deployedTo == Environment.PRODUCTION
-      val shouldBeVisible = subscriptions.exists(x => apis.contains(x.serviceName) && x.subscribed && isProduction)
+      val shouldBeVisible = subscriptions.exists(x => apis.contains(x.serviceName.value) && x.subscribed && isProduction)
```

Comment:
I think the list from config should be a list of serviceNames

Reply:
I agree, but how about merging this quickly to fix the issue in QA?

Reply:
Turning a strong type to a string feels like the wrong fix.

Reply:
Agreed it should be done right or we'll never do it - it's only QA

Reply:
Fixed